### PR TITLE
Updated documentation on UG and AboutUs

### PR DESCRIPTION
**Pull request by Jemereny**
AboutUs: Updated portfolio and github links

Userguide:
* Table of contents: Added features which will be implemented in version 2.0
* Intro: Added current version of application
* Features: Added Sort and Profile picture commands, updated documentation for existing commands which has updated features,
* Command Summary: Updated command summary to show new features and some examples
* Parameters: Updated to show which prefix accept which parameter

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -62,26 +65,34 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
+
+> ##Version 2.0## +
+Format: `add [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY] [w/WEBSITE] [t/TAG]...` +
+Shorthand: `a [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY] [w/WEBSITE] [t/TAG]...` +
+Only NAME is compulsory +
+* Name field *MUST* be included. +
+* Other fields are optional.
+* Birthday field is in DD/MM/YYYY
 
 TIPS:
```
Comment (chilipadiboy) [inline docs/UserGuide.adoc:85]: I think can remove this line if there are no tips left

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -298,32 +340,38 @@ There is no need to save manually.
...
-*Remark*^1^: r/ +
-*Website*: w/ +
-*Tags*^2^: t/ +
+*Name*: n/alphanumeric characters and spaces +
+*Phone Number*: p/at least 3 digits long +
+*Email*: e/ 2 alphanumeric/period strings separated by '@' +
+*Address*:  a/Address of person (can be any value) +
+*Birthday*: b/(DD/MM/YYYY) +
+*Remark*^1^: r/Can be any value or blank +
+*Website*: w/Should be a valid website +
```
Comment (chilipadiboy) [inline docs/UserGuide.adoc]: For now should add "beginning with http:// or https://  cause we don't have the auto append yet

**Review by chilipadiboy — CHANGES_REQUESTED** [review]

File: docs/UserGuide.adoc (hunk 1 above)
Comment (Jemereny) [reply docs/UserGuide.adoc:85]: which line?

Comment (xhsien) [reply docs/UserGuide.adoc:85]: There are lines below TIPS. I'll mark this as resolved.

**Pull request merged**